scripts/setup

```diff
@@ -171,6 +171,7 @@ eval ${PREFIX} invenio index queue purge delete
 set -e
 eval ${PREFIX} invenio index destroy --force --yes-i-know
 # Override index init to load templates before mapping
+# TODO: check if invenio index init --force works (to delete utils init --force)
```

Comment:
Can you check this now?

Reply:
It raises an `index_not_found_exception` error. Which explain why we use ours.